content/md/franklin-insights/franklin-insights-getting-started.md

```diff
@@ -0,0 +1,39 @@
+---
+id: franklin-insights-getting-started
+themes: franklin-insights
+title:  Franklin Insights **getting started**
+ee-only: true
+related: franklin-insights-mappings, franklin-insights-subscriptions
+---
+
+# Overview
+
+Welcome to Franklin’s world! Franklin is our new **product data intelligence assistant**, at your service :wink:
+
+Franklin helps Julia on her catalog enrichment mission and guides her towards creating a **more compelling product experience, faster than ever before**.
+
+:::info
+For now, Franklin can assist retailers who operate in the United States.
+:::
+# Franklin Activation
+
+The communication between the PIM and Franklin is done via API. PIM clients who purchase Franklin credits will receive an **activation code** (aka the API token). The activation code needs to be saved on the PIM on **System > Franklin Insights Connection** in order to activate communication between the PIM and Franklin.
```

Comment:
Should we really mention that it's working thanks to an API. Our readers being Julias this could look a bit too technical. What do you think?

Reply:
I'll remove this sentence